src/osbuild-manifests/platform.qemu.ipp.yaml

```diff
@@ -79,6 +81,8 @@ pipelines:
                 mpp-format-int: '{image.layout[''boot''].partnum}'
               target: /boot
   - name: qemu
+    build:
+      mpp-format-string: '{qemu_stage_buildroot}'
```

Comment:
I'm confused why we need this. Since the buildroot is scoped to the pipeline and here we're defining a different pipeline, isn't it going to use the host buildroot if we don't specify anything?

Reply:
yes. it will use the "host" buildroot if we don't specify anything, but I do like the explicitness that goes along with the comment where this is defined:

```
# For the qemu stages we'll use the host buildroot (COSA)
# because we definitely don't include qemu-img in CoreOS 
qemu_stage_buildroot: ""                                 

```

Reply:
OK to leave a comment but maybe let's make it clearer that that's the default anyway and we're just being explicit?